src/constants/index.ts

```diff
@@ -151,6 +151,6 @@ export const ADDRESS_ZERO = ethersContants.AddressZero;
 
 export const GANACHE_LOCAL_RPC_URL = 'http://localhost:8545';
 
-export const isDev = process.env.NODE_ENV === 'development';
```

Comment:
NODE_ENV does not exist in my .env nor my .env.example so switching to something that does